### ISSUE TITLE
Logo and Icon Pack

### ISSUE THREAD

**Issue by tophergates** [pet-heroes/pet-hero#6]
# Logo

Create a logo for Pet Hero. Example logos should be shared and discussed in the team discussion area before finalizing the logo and closing this issue. Preferably the logo will be high resolution and easily scaled to suit specific needs for web and (eventually) print media.

# Icon Pack

1. Create a 48x48 `favicon.ico` to be used as the favorite icon.

1. Create a pack of `.png` icons with transparent backgrounds to be used for (better) favicons in various browsers, home screen/mobile icons, etc. matching the sizes:

* favicon.png (16x16)
* favicon.png (32x32)
* mstile (150x150) - For Microsoft tiles
* apple-touch-icon (180x180)
* android-chrome (192x192)
* android-chrome (512x512)

**Comment by elioqoshi** [comment]
As said also on Open Source Design, I'd be happy to help. Any brief on the style of the logo?

**Comment by tophergates** [comment]
@elioqoshi thank you so much for reaching out to me. The logo I am envisioning, which will also be used for the icon(s), is a dog or a cat dressed up as a superhero standing in a confident superhero pose. I have placed a few examples in the mockups (#2) which we will not be able to use for the project due to licensing issues.

I'd be happy to speak more with you about the logo design if you are interested on our development discussion area for the project. Would you mind if I add you so we can chat some more about it?

**Comment by elioqoshi** [comment]
@tophergates I am up for a chat of course. Where do you prefer?

Regarding the logo: I think you are talking about a mascot, not a logo really. Having a fully fledged illustration in the logo is not suitable for the use cases a logo should serve. You'd want a similar approach as Duolingo I guess, where it has a simple wordmark as a logo and Duo, its mascot is used all over their mediums.

**Comment by tophergates** [comment]
@elioqoshi you're absolutely correct and Duolingo is a perfect example of what I am looking for. A nice logo with a mascot to solidify branding. 

For the logo text I am really only concerned with it having a comic-style to match the theme of the site and eventual mascot. @ryancmartin and I batted around the idea of it looking like a callout with the starburst background, like an old-school comic, similar to [this](https://images.cdn4.stockunlimited.net/clipart450/bang-comic-speech_1825098.jpg) but with 'Pet Hero!' instead of 'BANG'.

What are your thoughts on this? I know you cautioned on the design having too much of a comic book feel to it could distract from the overall message. Quite frankly, I have little to no eye for design, as you might be able to tell from my mockups. lol

**Comment by fabiyamada** [comment]
Hello, I dared to create this; a very simple and clean logo :)

![pet-hero_01](https://user-images.githubusercontent.com/33674731/37249878-46bd6bd4-24ad-11e8-8425-ab7ba55050ff.png)

More details here: https://steemit.com/utopian-io/@fabiyamada/logo-proposal-for-pet-hero

Thanks for your attention.
